slidechain/submit.go

```diff
@@ -17,8 +17,8 @@ import (
 	"github.com/interstellar/slingshot/slidechain/net"
 )
 
-// TODO: make this configurable.
-var blockInterval = 5 * time.Second
+// DefaultBlockInterval is the default duration between expected blocks on TxVM.
+const DefaultBlockInterval = 5 * time.Second
```

Comment:
Do we need to export this?

Reply:
also, if we pass `blockInterval` as a parameter into `newCustodian` we could probably get rid of this constant (& just define a block interval in tests)

Reply:
Discussed offline, but for posterity -- the current approach helps ensure the same default block interval in `slidechaind` and `slidechain_test` and makes future changes slightly easier, so we'll keep it over hand-defining durations in different parts of the codebase.